### ISSUE TITLE
inverting the image intensity as a preprocessing step 

### ISSUE THREAD

**Issue by VolkerH** [MouseLand/cellpose#26]
I just wanted to suggest that it may be worth mentioning in the Readme that inverting the image intensity may give better results with histology staining (which is often dark on light background) when using the pre-trained model.  

I assume that with enough training on such examples the preprocessing step would become unncessary, but if one wants a quick result without retraining this is helpful. 
Maybe retraining with image inversion as a data augmentation strategy may also help (unless this is already done ... in which case I do not understand why I would see a difference).

This is somehting that has been reported for Stardist and I also observed this with Cellpose. I 
assume the underlying cause is that the pre-trained networks have seen more light on dark examples.
 
I have not done proper benchmarking, but below is some anecdotal evidence (note that these are not perfectly overlapping crops and I may have used slightly different size estimates):

![image](https://user-images.githubusercontent.com/2210044/75736392-35871200-5d51-11ea-8818-4773b5eb600b.png)

after colour deconvoution and inversion
![image](https://user-images.githubusercontent.com/2210044/75736413-4172d400-5d51-11ea-8820-d8b2fe7411a7.png)

**Comment by carsen-stringer** [comment]
i've added this option to the gui, please let me know if you are having issues I will close for now

**Comment by carsen-stringer** [comment]
I also will note that we ONLY trained the nucleus model on light on dark images (we inverted the images in the training and test set for all models that we trained including ours), so inverting images like yours is a good idea

**Comment by VolkerH** [comment]
Thanks for implementing this. Will give it a try next week.
I don't quite understand your last statement as I don't know what "all models that we trained  including ours" refers to.

Are you saying that all models other than the nucleus model have been trained using intensity inversion as a data augmentation step? Meaning that model that detects whole cells should not require intensity inversion? If so, that would be inconsisten with my observations as I noticed that
inverting the input image made difference in results for both models.
